brave-grpc/src/main/java/com/github/kristofa/brave/grpc/BraveGrpcServerInterceptor.java

```diff
@@ -90,7 +93,14 @@ public String getSpanName() {
 
         @Override
         public Collection<KeyValueAnnotation> requestAnnotations() {
-            return Collections.emptyList();
+            SocketAddress socketAddress = call.attributes().get(ServerCall.REMOTE_ADDR_KEY);
+            if (socketAddress != null) {
+                KeyValueAnnotation remoteAddrAnnotation = KeyValueAnnotation.create(
```

Comment:
in zipkin, it would be more normal to add a binary annotation of type "ca" (client address). This is done via `ServerTracer.setServerReceived(int ipv4, int port, @Nullable String clientService)`, though maybe it isn't easy to get at here..